keanu-project/src/main/java/io/improbable/keanu/tensor/TensorShape.java

```diff
@@ -1,5 +1,7 @@
 package io.improbable.keanu.tensor;
 
+import io.improbable.keanu.tensor.dbl.DoubleTensor;
```

Comment:
Unused import

Reply:
You can configure IntelliJ to remove these automatically: Editor - General - Auto Import

Reply:
Wow I didn't know you could get IntelliJ to just magically optimize imports on the fly!